modules/consignment-api/variables.tf

```diff
@@ -6,4 +6,5 @@ variable "public_subnets" {}
 variable "private_subnets" {}
 variable "database_availability_zones" {}
 variable "region" {}
-variable "db_migration_sg" {}
\ No newline at end of file
+variable "db_migration_sg" {}
+variable "environment_full" {}
```

Comment:
Could this be changed to environment_full_name just to make it clear what it is?

Reply:
I've changed the name here and for the map as well.